lib/Resource.js

```diff
@@ -38,7 +38,14 @@ var Resource = function(options) {
   this.search = options.search;
   this.sort = options.sort;
 
+  this.associationOptions = {
```

Comment:
this should be added to the ```mergeDefaults``` above, below ```include```. Then you don't have to use _.extend, and we can more easily track all the defaults for documentation purposes